front/src/modules/pathfinding/components/Itinerary/DisplayViasV2.tsx

```diff
@@ -1,14 +1,87 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { XCircle } from '@osrd-project/ui-icons';
 import cx from 'classnames';
 import type { Position } from 'geojson';
 import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
 import { useSelector } from 'react-redux';
 
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
 import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
+import type { PathStep } from 'reducers/osrdconf/types';
 import { useAppDispatch } from 'store';
+import { useDebounce } from 'utils/helpers';
 import { formatUicToCi } from 'utils/strings';
+import { ISO8601Duration2sec, formatDurationAsISO8601 } from 'utils/timeManipulation';
+
+const StopDurationSelector = ({
+  via,
+  focusedViaId,
+  setFocusedViaId,
+}: {
+  via: PathStep;
+  focusedViaId?: string;
+  setFocusedViaId: (focusedViaIndex?: string) => void;
+}) => {
+  const dispatch = useAppDispatch();
+  const { updateViaStopTimeV2 } = useOsrdConfActions();
+
+  const currentStopTime = via.stop_for ? ISO8601Duration2sec(via.stop_for) : 0;
+
+  const [stopTime, setStopTime] = useState(currentStopTime);
+  const debouncedStopTime = useDebounce(stopTime, 1000);
```

Comment:
I think the debounce time is a little to short. If I think more than 1sec while typing a new duration, the component will be rerender and I lose the focus in the input.

Reply:
I change for 2s, do you think it's enough ?